agent-codeless/src/main/java/com/microsoft/applicationinsights/agentc/internal/diagnostics/MachineNameFinder.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.applicationinsights.agentc.internal.diagnostics;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class MachineNameFinder extends CachedDiagnosticsValueFinder {
+    public static final String PROPERTY_NAME = "MachineName";
+
+    @Nullable
+    @Override
+    protected String populateValue() {
```

Comment:
is WEBSITES_SITENAME is covered somewhere else?

Reply:
Yes, here: https://github.com/microsoft/ApplicationInsights-Java/blob/codeless/agent-codeless/src/main/java/com/microsoft/applicationinsights/agentc/internal/diagnostics/SiteNameFinder.java